components/brave_vpn/brave_vpn_service.cc

```diff
@@ -585,14 +585,6 @@ void BraveVpnService::LoadPurchasedState(const std::string& domain) {
     return;
   }
 
-#if !BUILDFLAG(IS_ANDROID)
-  if (!IsNetworkAvailable()) {
```

Comment:
I think removing this code is great 👍 `LoadPurchasedState` is called when person is clicking on `VPN` button... and if they didn't have network connectivity (briefly, for example), this code would be letting the connection go into FAILED (when no connect ever was made by user).

Nice! 😄 👍

Reply:
As we cache expiration date, browser will enter to purchased state even network is disabled.
Then, user can get connect failure when user try to connect.
Or it's already expired, browser will ask to refresh skus credential and skus service will make use not purchased state.
So we could get proper state w/o this early checking.